[BUG] Translations / error handling

### Pre-flight checklist

- [x] I have searched [existing issues](https://github.com/mauriceboe/TREK/issues) and this bug has not been reported yet
- [x] I am running the latest available version of TREK

### TREK version

2.9.11

### Describe the bug

Testing the newly configured SMTP settings with the test button says: **SMTP not configured** which is misleading because my config was wrong but not, not configured. Maybe you could give the backend error from the logs directly to the user:

`[ERROR] 2026-04-09T15:14:35 Email send failed to=user@xyz.de: getaddrinfo ENOTFOUND ssl.xyz.net`


And a translation issue. All test buttons in German say **Test-E-Mail senden** even for admin webhooks.

And I can also not open or change the normal webhook settings anymore.

Backup timestamp is also buggy see the screenshot.

### Steps to reproduce

Go to admin settings under the tab notifications.

### Expected behavior

-

### Deployment method

Unraid template

### Host OS

Unraid 7.2.3

### Accessing TREK from

Desktop browser

### Browser (if applicable)

_No response_

### Relevant logs or error output

```shell

```

### Screenshots

<img width="1458" height="322" alt="Image" src="https://github.com/user-attachments/assets/707bc543-a538-4fb8-a7cd-e2b7197e21d0" />

### Additional context

_No response_

--- a/client/src/pages/AdminPage.tsx
+++ b/client/src/pages/AdminPage.tsx
@@ -1353,7 +1353,7 @@ export default function AdminPage(): React.ReactElement {
                       disabled={!smtpValues.admin_webhook_url?.trim()}
                       className="px-4 py-2 border border-slate-300 text-slate-700 rounded-lg text-sm font-medium hover:bg-slate-50 transition-colors disabled:opacity-40"
                     >
-                      {t('admin.smtp.testButton')}
+                      {t('admin.notifications.testWebhook')}
                     </button>
                   </div>
                 </div>

--- a/server/src/services/backupService.ts
+++ b/server/src/services/backupService.ts
@@ -117,7 +117,7 @@ export function listBackups(): BackupInfo[] {
         filename,
         size: stat.size,
         sizeText: formatSize(stat.size),
-        created_at: stat.birthtime.toISOString(),
+        created_at: stat.mtime.toISOString(),
       };
     })
     .sort((a, b) => new Date(b.created_at).getTime() - new Date(a.created_at).getTime());

--- a/server/src/services/notifications.ts
+++ b/server/src/services/notifications.ts
@@ -399,9 +399,24 @@ export async function sendWebhook(url: string, payload: { event: string; title: 
 }
 
 export async function testSmtp(to: string): Promise<{ success: boolean; error?: string }> {
+  if (!getSmtpConfig()) return { success: false, error: 'SMTP not configured' };
   try {
-    const sent = await sendEmail(to, 'Test Notification', 'This is a test email from TREK. If you received this, your SMTP configuration is working correctly.');
-    return sent ? { success: true } : { success: false, error: 'SMTP not configured' };
+    const config = getSmtpConfig()!;
+    const skipTls = process.env.SMTP_SKIP_TLS_VERIFY === 'true' || getAppSetting('smtp_skip_tls_verify') === 'true';
+    const transporter = nodemailer.createTransport({
+      host: config.host,
+      port: config.port,
+      secure: config.secure,
+      auth: config.user ? { user: config.user, pass: config.pass } : undefined,
+      ...(skipTls ? { tls: { rejectUnauthorized: false } } : {}),
+    });
+    await transporter.sendMail({
+      from: config.from,
+      to,
+      subject: 'TREK — Test Notification',
+      text: 'This is a test email from TREK. If you received this, your SMTP configuration is working correctly.',
+    });
+    return { success: true };
   } catch (err) {
     return { success: false, error: err instanceof Error ? err.message : 'Unknown error' };
   }

--- a/server/tests/unit/services/backupService.test.ts
+++ b/server/tests/unit/services/backupService.test.ts
@@ -580,7 +580,7 @@ describe('BACKUP-041 listBackups', () => {
     fsMock.readdirSync.mockReturnValue(['backup-2026-01-01T00-00-00.zip']);
     fsMock.statSync.mockReturnValue({
       size: 1024,
-      birthtime: new Date('2026-01-01T00:00:00Z'),
+      mtime: new Date('2026-01-01T00:00:00Z'),
     });
 
     const result = listBackups();
@@ -599,9 +599,9 @@ describe('BACKUP-041 listBackups', () => {
     ]);
     fsMock.statSync.mockImplementation((p: string) => {
       if (String(p).includes('2026-01-01')) {
-        return { size: 512, birthtime: new Date('2026-01-01T00:00:00Z') };
+        return { size: 512, mtime: new Date('2026-01-01T00:00:00Z') };
       }
-      return { size: 2048, birthtime: new Date('2026-06-01T00:00:00Z') };
+      return { size: 2048, mtime: new Date('2026-06-01T00:00:00Z') };
     });
 
     const result = listBackups();
@@ -619,7 +619,7 @@ describe('BACKUP-041 listBackups', () => {
     ]);
     fsMock.statSync.mockReturnValue({
       size: 1024,
-      birthtime: new Date('2026-01-01T00:00:00Z'),
+      mtime: new Date('2026-01-01T00:00:00Z'),
     });
 
     const result = listBackups();

